components/ModEditSubmissionForm.vue

```diff
@@ -292,217 +293,245 @@
+                </button>
             </div>
-            <h2
-                :id="ModSubmissionLeftNavbarSectionIDs.HealthcareProfessionalMedicalInfo"
-                class="submission-form-section
+            <div class="flex flex-col">
+                <h2
+                    :id="ModSubmissionLeftNavbarSectionIDs.HealthcareProfessionalMedicalInfo"
+                    class="submission-form-section
                     my-3.5 text-start text-primary-text text-2xl font-bold font-sans leading-normal"
-            >
-                {{ $t('modSubmissionForm.healthcareProfessionalMedicalInfoHeading') }}
-            </h2>
-            <label
-                for="Accepted Insurances"
-                class="my-2 text-primary-text text-sm font-bold font-sans"
-            >
-                {{ $t("modSubmissionForm.selectInsurances") }}
-            </label>
-            <ModSearchBar
-                v-model="submissionFormFields.healthcareProfessionalAcceptedInsurances"
-                data-testid="submission-form-accepted-insurances"
-                :place-holder-text="$t('modSubmissionForm.placeholderTextAcceptedInsurances')"
-                :no-match-text="$t('modSubmissionForm.noInsurancesWereFound')"
-                :fields-to-display-callback="insurancesToDisplayCallback"
-                @search-input-change="handleInsuranceInputChange"
-            />
-            <ol class="list-disc text-primary-text/60 font-semibold my-2 px-2">
-                <li
-                    v-for="insurance in submissionFormFields.healthcareProfessionalAcceptedInsurances"
-                    :key="`accepted-${insurance}`"
-                    class="py-1"
                 >
-                    {{ insurance }}
-                </li>
-            </ol>
-            <label
-                for="degrees"
-                class="my-2 text-primary-text text-sm font-bold font-sans"
-            >
-                {{ $t("modSubmissionForm.selectDegrees") }}
-            </label>
-            <ModSearchBar
-                v-model="submissionFormFields.healthcareProfessionalDegrees"
-                data-testid="submission-form-degrees"
-                :place-holder-text="$t('modSubmissionForm.placeholderTextDegrees')"
-                :no-match-text="$t('modSubmissionForm.noDegreesWereFound')"
-                :fields-to-display-callback="degreesToDisplayCallback"
-                @search-input-change="handleDegreeInputChange"
-            />
-            <ol class="list-disc text-primary-text/60 font-semibold my-2 px-2">
-                <li
-                    v-for="degree in submissionFormFields.healthcareProfessionalDegrees"
-                    :key="`current-${degree}`"
-                    class="py-1"
+                    {{ $t('modSubmissionForm.healthcareProfessionalMedicalInfoHeading') }}
+                </h2>
+                <label
+                    for="Accepted Insurances"
```

Comment:
Do we need this `for` attribute? If so, I think the value should be lowercase to match the corresponding element id like:

```suggestion
                    for="accepted-insurances"
```

[MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/for)

Reply:
it is for screen readers. So as long as the casing matches what it is for in the name of the input

Reply:
So the `ModSearchBar` input should take in the prop name. This can be done as a different issue. Along with allowing for the wildcard that when it is focused if nothing is typed it shows all.